torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1616,6 +1601,15 @@ void InitXlaModuleBindings(py::module m) {
         [](at::Tensor& self, const at::Tensor& source) -> at::Tensor& {
           return XLANativeFunctions::set_(self, source);
         });
+  m.def("_get_all_reduce_token",
```

Comment:
I think it is better to call it `_get_cc_token` or `_get_xla_token`, although it is currently only for `all_reduce`. We can also do this after we convert second cc op to use cpp token

Reply:
I'm just following the traditional in the python layer where it's named as all_reduce_token. haha.